mungegithub/publisher/pv.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
```

Comment:
Why do we need the pv? I thought the pvc controller will create the pv automatically.

Reply:
The Makefile wants it. I only adapted the code to work with the default deployment mechanism.